src/html/components/mm/templates/content-accordion.handlebars

```diff
@@ -0,0 +1,67 @@
+{{> header}}
+<main id='contentTemplate'> 
+    <nav class="content__breadcrumb">
+        <ul class="row">
+            <li><a href="#">Templates</a></li>
+            <li><a href="#">Accordion examples</a></li>
+        </ul>
+    </nav>    
+    <section class="container container--bg-white content__content">
+        <article  class="row">
+            {{> content-side}} 
+            <article class="content__main">
+                <h1>Accordion</h1>
+                <section class="accordion">
+                    {{#each accordions}}
+                        {{#if this.striped}}
+                            {{printHeader this.header this.striped}}
+                            <h3>{{this.header}}</h3>
+                            {{#if this.color}}
+                            <h4><button class="accordion__heading accordion__heading--{{this.color}} accordion__heading--{{this.color}}-striped" aria-expanded="false" id="accordion{{this.id}}id" aria-controls="content-{{this.id}}-1">Unordered list<i aria-hidden="true" class="fa fa-plus-square"></i><i aria-hidden="true" class="fa fa-minus-square"></i></button></h4>
```

Comment:
The accordion headers shouldn't use an `<h4>` tag, they should just be buttons. 

If this accordion was included on a page that didn't have any h2 or h3, than it wouldn't make sense.